### ISSUE TITLE
[Docs] Venv guide - using `%userprofile%` is unsafe when username has spaces

### ISSUE THREAD

**Issue by Jackenmen** [Cog-Creators/Red-DiscordBot#3411]
Example issue:
```
C:\Users\Space Ranger>py -3.8 -m venv %userprofile%\redenv
Error: [Errno 13] Permission denied: 'C:\\Users\\Space\\pyvenv.cfg'
```

**Comment by mikeshardmind** [comment]
Recommended route: Update venv guide to continue using PowerShell from the install guide, use `$env:USERPROFILE`

Other proper escaping or argument quoting of this is also acceptable

**Comment by palmtree5** [comment]
quotes should do the trick but I'm inclined to agree with @mikeshardmind that we should probably update this to use powershell since that's our recommended install method on Windows

**Comment by Drapersniper** [comment]
Hmm a lot of users will end up installing red as admin if this is the case though as when they open power shell they are told to open it as admin

**Comment by mikeshardmind** [comment]
They are also given a command which exits powershell. We shouldn't make our directions worse for people reading them, just because people not reading them could make a mistake.

**Comment by Jackenmen** [comment]
Fixed with #3495
